packages/rancher-gke-operator/charts/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+annotations:
```

Comment:
Do we want a copy of this chart in the `packages/` directory?

Reply:
:raised_eyebrow: not sure how this ended up here